### PR TITLE
(MODULES-2634) Try / Catch Test Updates

### PR DESCRIPTION
**Pull request by Iristyle**
```
Finished in 4 minutes 31.7 seconds (files took 2 minutes 24.6 seconds to load)
27 examples, 0 failures
```

### CODE REVIEW

**Comment by Iristyle** [comment]
@puppetlabs/windows this is ready to go

File: spec/spec_helper_acceptance.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@
     shell("/bin/touch #{default['puppetpath']}/hiera.yaml")
   end
   c.after :suite do
-    absent_files = 'file{["c:/services.txt","c:/process.txt"]: ensure => absent }'
+    absent_files = 'file{["c:/services.txt","c:/process.txt","c:/try_success.txt","c:/catch_shouldntexist.txt","c:/try_shouldntexist.txt","c:/catch_success.txt"]: ensure => absent }'
```
Comment (ferventcoder) [inline spec/spec_helper_acceptance.rb]: That's something I didn't realize - that it would not clean up those files by after each test.

**Comment by ferventcoder** [comment]
Running this through adhoc now.

**Comment by ferventcoder** [comment]
👍

**Pull request merged**